### ISSUE TITLE
AuthAdapters cannot be defined by CLI

### ISSUE THREAD

**Issue by dblythy** [parse-community/parse-server#8665]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

When starting Parse Server via CLI and with JSON, it's not possible to define auth adapters

### Steps to reproduce

```js
{
  "appName": "test",
  "appId": "test",
  "masterKey": "test",
  "logLevel": "error",
  "auth": {
    "facebook": {
      "appIds": "test"
    }
  }
}

```

### Actual Outcome

`Error: [object Object] should be a comma separated string or an array`

### Environment

Server
- Parse Server version: `alpha`
- Operating system: `macos`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `localhost`

**Comment by dblythy** [comment]
First reported by @FransGH https://github.com/parse-community/parse-server/pull/7953#issuecomment-1606053116

**Comment by FransGH** [comment]
@dblythy thx for the quick response/fix!

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.3.0-alpha.4](https://github.com/parse-community/parse-server/releases/tag/6.3.0-alpha.4)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.4.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/6.4.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.4.0-alpha.1](https://github.com/parse-community/parse-server/releases/tag/6.4.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.4.0](https://github.com/parse-community/parse-server/releases/tag/6.4.0)
